target/linux/ipq40xx/patches-4.19/304-spi-nor-enable-4B-opcodes-for-w25q256jv.patch

```diff
@@ -0,0 +1,12 @@
+--- a/drivers/mtd/spi-nor/spi-nor.c
++++ b/drivers/mtd/spi-nor/spi-nor.c
+@@ -1253,7 +1253,8 @@ static const struct flash_info spi_nor_i
+ 	{ "w25q80", INFO(0xef5014, 0, 64 * 1024,  16, SECT_4K) },
+ 	{ "w25q80bl", INFO(0xef4014, 0, 64 * 1024,  16, SECT_4K) },
+ 	{ "w25q128", INFO(0xef4018, 0, 64 * 1024, 256, SECT_4K) },
+-	{ "w25q256", INFO(0xef4019, 0, 64 * 1024, 512, SECT_4K | SPI_NOR_DUAL_READ | SPI_NOR_QUAD_READ) },
```

Comment:
This needs to go into separate commit with proper description, ideally sent upstream...

Reply:
Well, as far as I understand the reason why this was never upstreamed was that regular w25q256 and w25q256 share the same JEDEC ID so you really cant figure out if its old or JV revision

Reply:
Separate commit please.

Reply:
5.4 can't come soon enough...